### DISCUSSION TITLE
期望能够为OpenAI的不同模型设置独立的快捷键

### DISCUSSION THREAD

**Discussion by a90091343** [immersive-translate/immersive-translate#2197] [Ideas]
### 新功能 | New Feature

1.期望能够为OpenAI的不同模型设置独立的快捷键

### 为什么您想要添加这些功能 | Why you want to add these features

1.众所周知，gpt-4o和gpt-4-mini，无论价格和翻译速度都相差甚大；且mini模型的翻译能力经测试，准确度和语句的流畅度是不如4o的；
2.有些场景的内容不是很重要，只需速览，适合mini模型；有些内容如论文很重要，需要精读，适合4o；
3. 现在要切换两者，每次都需要到设置里更改模型；如果有独立的快捷键，就方便许多了。

### 提议 | Proposal

谢谢 immersive-translate 的开发者们！你们造福了广大科研人！

**Comment by theowenyoung** [comment]
你可以在翻译服务里，添加一个额外的自定义openai翻译服务，为那个翻译服务设置不同的模型，然后再在快捷键设置里设置快捷键。

**Reply by a90091343** [reply]
成功了，谢谢你！
